transformer/testdata/search_example.json

```diff
@@ -79,7 +79,7 @@
                         },
                         "highlight": {
                             "description.title": [
-                                "<strong>Housing</strong>"
+                                "<em>Housing</em>"
```

Comment:
`<em class="highlight">` is the class used in the design system (https://ons-design-system.netlify.app/patterns/see-important-information/)